common/shlibs

```diff
@@ -4147,8 +4147,8 @@ libCore.so root-6.24.06_1
 libCore.so qtcreator-7.0.2_1
 liblowdown.so lowdown-0.11.2_1
 libjaylink.so.0 libjaylink-0.2.0_1
-libnvidia-container.so.1 libnvidia-container-1.10.0_1
```

Comment:
if the soname didn't change, this should not be bumped (it's the minimum version that provides the shlib)

Reply:
thanks, fixed